src/test/java/org/reactome/OPTIONALSUBDOMAINHERE/TestMain.java

```diff
@@ -0,0 +1,33 @@
+package org.reactome.OPTIONALSUBDOMAINHERE;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
```

Comment:
Simply assuming that other developers want to use hamcrest? ;)

Reply:
Giving a gentle nudge of encouragement :)

Reply:
From what I've read and experienced, I think Hamcrest offers a lot of advantages over native JUnit asserts.

https://stackoverflow.com/questions/1701113/why-should-i-use-hamcrest-matcher-and-assertthat-instead-of-traditional-assert

Did you have any concerns over using Hamcrest?

Reply:
I've never had an issue with JUnit's own assertion functions. Until I do, I'll probably stick with them. From what I can tell, hamcrest is little more than syntactic sugar, but with no significantly better functionality.

Reply:
I also do think that since this is a template, it should not be including dependencies, except for what is relevant to the template's context (such as `reactome-base`).

Reply:
I do think we should explicitly state that we are _encouraging_ Hamcrest over Junit in our testing, and that the reasons are for increased legibility and error handling.

Reply:
I'll add some comments/documentation to make it clear that we're not forcing the use of Hamcrest and just recommending it based on those points.